### DISCUSSION TITLE
图上面有大量的空白

### DISCUSSION THREAD

**Discussion by pkeke** [note286/xduts#48] [Q&A] [answered]
图片在新的一页，但并不在顶部，上面有大量的空白。图片本身上面并没有大量空白。
![image](https://user-images.githubusercontent.com/44956389/219937841-c937c6d4-ed3c-432d-80df-f3534603fddf.png)
latex代码如下：
```
\begin{figure}[htbp]
	\centering
	% Requires \usepackage{graphicx}
	\includegraphics[]{figures/4-1.pdf}
	\caption{123}
	\label{4-1}
\end{figure}
```

**Comment by note286** [comment, marked as answer]
先写文字，后面有文字自然就浮动上去了。

**Reply by pkeke** [reply]
已解决，谢谢！
